addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17812,7 +17818,29 @@ msgctxt "#36614"
 msgid "Show add-ons that are currently running in the background."
 msgstr ""
 
-#empty strings from id 36615 to 36899
+#: system/settings/settings.xml
+msgctxt "#36615"
+msgid "Unknown sources"
+msgstr ""
+
+#. Description of setting "System -> Add-ons -> Unknown sources" with label #36615
+#: system/settings/settings.xml
+msgctxt "#36616"
+msgid "Allow installation of add-ons from unknown sources."
+msgstr ""
+
+#: xbmc/addons/GUIWindowAddonBrowser.cpp
+msgctxt "#36617"
+msgid "For security, installation of add-ons from unknown sources is disabled."
+msgstr ""
+
+#. Text in confirm dialog when enabling setting "System -> Add-ons -> Unknown sources"
+#: xbmc/addons/AddonSystemSettings.cpp
+msgctxt "#36618"
+msgid "Add-ons will be given access to personal data stored on this device. By allowing, you agree that you are solely responsible for any loss of data, unwanted behaviour, or damage to your device that may result from using these add-ons. Proceed?"
```

Comment:
I found that part actually quite nice as it gives an extra emphasis. Also, I think we should probably mention that not only the data of this particular device is exposed, python add-ons can also access any device within the network if not protected.

What about a little rephrased version like:
`Add-ons will have access to all unprotected data on this device, which includes your personal data, and exposed data on your local network. By allowing, you agree that you are solely responsible for any loss of data, unwanted behavior or damage to your device that these add-ons may cause. Proceed?`